include/circt/Dialect/Calyx/CalyxLoweringUtils.h

```diff
@@ -424,6 +424,15 @@ class ComponentLoweringStateInterface {
     Block *body = component.getBodyBlock();
     builder.setInsertionPoint(body, body->begin());
     auto name = TLibraryOp::getOperationName().split(".").second;
+    if constexpr (TLibraryOp::template hasTrait<FloatingPoint>()) {
+      auto standard = TLibraryOp::getFloatingPointStandard();
+      if (standard == FloatingPointStandard::IEEE754) {
```

Comment:
I think a switch would be appropriate here, so that that if someone adds another case the compiler will complain if we don't update this.
```suggestion
      switch(TLibraryOp::getFloatingPointStandard()) {
        case FloatingPointStandard::IEEE754:
          ...
      }
```